.circleci/config.yml

```diff
@@ -297,14 +297,14 @@ jobs:
 
   smoketest_dev:
     docker:
-      - image: 'cypress/base:latest'
+      - image: 'cypress/base:16.5.0'
```

Comment:
The latest tag for cypress/base hasn't been updated in a year (and therefore clearly isn't the latest)

Reply:
I've updated this since it was out of step with the cypress test dependencies